buildsettings.json

```diff
@@ -1,7 +1,7 @@
 {
-    "version": "1.8.1",
+    "version": "1.9.1",
     "avi": {
-        "maxVersion": "22.1.1",
+        "maxVersion": "22.1.3",
```

Comment:
Please make identical changes in this doc also. https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tree/master/docs#ako-compatibility-guide